src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card"data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+      class="card__photo"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h1 class="card__code">
+        Product code: 195434
+      </h1>
```

Comment:
`<h1>` is a tag for main title of the page and can be used only once. Just `<p>` fits better here